openupgradelib/openupgrade.py

```diff
@@ -799,29 +799,60 @@ def float_to_integer(cr, table, field):
 
 
 def map_values(
-        cr, source_column, target_column, mapping,
-        model=None, table=None, write='sql'):
+        cr, source_column, target_column, mapping, xml_mapping=False,
```

Comment:
Did I miss something? I wasn't aware of that keyword arguments can be passed positionally... Is that so?